src/sites/twitch-twilight/modules/host-options.vue

```diff
@@ -0,0 +1,176 @@
+				<span class="tw-button__text">Add To Auto Host</span>
+			</button>
+		</header>
+	</div>
+	<div class="tab overflow-hidden"
+		v-show="activeTab === 'settings'"
+		v-bind:class="{ active: activeTab === 'settings'}">
+		<section class="border-t full-width full-height">
+			<main data-simplebar="init" class="flex-grow-1 scrollable-area">
+				<div class="simplebar-track vertical" style="visibility: hidden;">
+					<div class="simplebar-scrollbar"></div>
+				</div>
+				<div class="simplebar-track horizontal" style="visibility: hidden;">
+					<div class="simplebar-scrollbar"></div>
+				</div>
+				<div class="simplebar-scroll-content" style="padding-right: 17px; margin-bottom: -34px;">
+					<div class="simplebar-content ffz-host-options-sortable" style="padding-bottom: 17px; margin-right: -17px;">
+						<div class="pd-1">
+							<div class="ffz--widget ffz--checkbox">
+								<div class="flex align-items-center">
+									<input type="checkbox" class="tw-checkbox__input"
+										id="autoHostSettings:enabled"
+										data-setting="enabled"
+										:checked="autoHostSettings.enabled"
+										@change="updateCheckbox">
+									<label for="autoHostSettings:enabled" class="tw-checkbox__label">
+										Auto Hosting
+									</label>
+								</div>
+								<section class="c-text-alt-2" style="padding-left: 2.2rem;">
+									Toggle all forms of auto hosting: teammates, host list, and similar channels.<br>
+									<a href="https://blog.twitch.tv/grow-your-community-with-auto-hosting-e80c1460f6e1" target="_blank" rel="noopener">Learn More</a>
+								</section>
+							</div>
+							<div class="ffz--widget ffz--checkbox">
+								<div class="flex align-items-center">
+									<input type="checkbox" class="tw-checkbox__input"
+										id="autoHostSettings:team_host"
+										data-setting="team_host"
+										:checked="autoHostSettings.team_host"
+										@change="updateCheckbox">
+									<label for="autoHostSettings:team_host" class="tw-checkbox__label">
+										Team Hosting
+									</label>
+								</div>
+								<section class="c-text-alt-2" style="padding-left: 2.2rem;">
+									Automatically host random channels from your team when you're not live.<br>
+									Team channels will be hosted before any channels in your host list.
+								</section>
+							</div>
+							<div class="ffz--widget ffz--checkbox">
+								<div class="flex align-items-center">
+									<input type="checkbox" class="tw-checkbox__input"
+										id="autoHostSettings:vodcast_hosting"
+										data-setting="deprioritize_vodcast"
+										:checked="!autoHostSettings.deprioritize_vodcast"
+										@change="updateCheckbox">
+									<label for="autoHostSettings:vodcast_hosting" class="tw-checkbox__label">
+										Vodcast Hosting
+									</label>
+								</div>
+								<section class="c-text-alt-2" style="padding-left: 2.2rem;">
+									Include Vodcasts in auto host.<br>
```

Comment:
Needs translation support. Why is this going on a new line? It doesn't seem like it needs it.